### PR TITLE
#143/Fix edit user page

### PR DESCRIPTION
**Pull request by fizcamposancos**
I merged the Edit User Page before pushing the modifications from your comments. Sorry,

### CODE REVIEW

**Comment by Damrod** [comment]
I think we should revert latest master commit, otherwise you have hundreds of merge conflicts (because the previous one was squashed)

**Comment by Damrod** [comment]
I think we should revert latest master commit, otherwise you have hundreds of merge conflicts (because the previous one was squashed)

File: webapp/src/pages/EditUserPage/EditUserPage.css [hunk 1]
```diff
@@ -24,31 +24,31 @@
 
 @media all and (min-width: 768px) {
   .edit-user-page {
-    display: grid;
     grid-template-columns: repeat(var(--column-number), 1fr);
-    grid-template-rows: auto 1fr auto;
-    grid-row-gap: 8rem;
+    grid-template-rows: auto 20% auto 1fr aut;
```
Comment (Damrod) [inline webapp/src/pages/EditUserPage/EditUserPage.css:28]: aut**o**

Comment (Damrod) [inline webapp/src/pages/EditUserPage/EditUserPage.css:28]: aut**o**

Comment (Damrod) [reply webapp/src/pages/EditUserPage/EditUserPage.css:28]: this should do it 
```
diff --git a/webapp/src/pages/EditUserPage/EditUserPage.css b/webapp/src/pages/EditUserPage/EditUserPage.css
index 3171ef7..266af01 100644
--- a/webapp/src/pages/EditUserPage/EditUserPage.css
+++ b/webapp/src/pages/EditUserPage/EditUserPage.css
@@ -25,7 +25,8 @@
 @media all and (min-width: 768px) {
   .edit-user-page {
     grid-template-columns: repeat(var(--column-number), 1fr);
-    grid-template-rows: auto 20% auto 1fr aut;
+    grid-template-rows: auto 20% auto 1fr auto;
+    row-gap: 0;
   }
   .edit-user-page .header {
     grid-row: 1;
@@ -36,14 +37,15 @@
     grid-column: 2/6;
     grid-row: 3/4;
     align-self: start;
+    justify-self: flex-end;
   }
   .edit-user-form {
     grid-row: 3/4;
     grid-column: 7/12;
   }
   .edit-user-form-button {
-    grid-row: 4/5;
-    grid-column: 5/9;
+    grid-row: 5/6;
+    grid-column: 1 / var(--column-end);
   }
 }
```

**Review by Damrod — APPROVED** [review]
feel free to merge when you take in my changes!

File: webapp/src/pages/EditUserPage/EditUserPage.css (hunk 1 above)
Comment (fizcamposancos) [reply webapp/src/pages/EditUserPage/EditUserPage.css]: Done

**Pull request merged**